migrations/20200304235514-add-item-associations.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    /*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      return queryInterface.createTable('users', { id: Sequelize.INTEGER });
+    */
+    return queryInterface.sequelize.transaction(function(transaction) {
```

Comment:
This file is a new MIGRATION that changes the database structure a bit- I had to rename some of the Item columns to match what Sequelize expect...

categoryId -> CategoryId
restaurantId -> RestaurantId